src/view/components/alert.styl

```diff
@@ -20,10 +20,6 @@
   border-color: var(--spectrum-semantic-negative-color-default);
 }
 
-.Alert-title {
-  margin: 0
-}
```

Comment:
The styles Spectrum CSS provides for our Heading component specify no margin, which allowed me to remove this.